### PR TITLE
Add Icons to User Team. Need to reload mysql database.

### PR DESCRIPTION
**Pull request by PossiblyJustun**

### CODE REVIEW

File: backend/src/main/java/com/example/backend/UserTeams/UserTeamController.java [hunk 1]
```diff
@@ -20,7 +20,7 @@ public class UserTeamController {
     public ResponseEntity<String> createUserTeam(@PathVariable Integer userId, @PathVariable String teamName) {
         try {
             Connection connection = DriverManager.getConnection(Constants.url, Constants.username, Constants.password);
-            String insertQuery = "INSERT INTO UserTeams (team_name) VALUES('" + teamName + "')";
+            String insertQuery = "INSERT INTO UserTeams (team_name, icon) VALUES('" + teamName + "', 'Icon1.png')";
```
Comment (ad-world) [inline backend/src/main/java/com/example/backend/UserTeams/UserTeamController.java:23]: Is this supposed to be icon1.png? Like is this a default value or something? Or should it be something g else

**Review by ad-world — COMMENTED** [review]
Left a comment

File: backend/src/main/java/com/example/backend/UserTeams/UserTeamController.java (hunk 1 above)
Comment (PossiblyJustun) [reply backend/src/main/java/com/example/backend/UserTeams/UserTeamController.java:23]: > Is this supposed to be icon1.png? Like is this a default value or something? Or should it be something g else

Yeah I just use it as the default value. Or else the icon is empty.

**Review by Zinka010 — APPROVED** [review]
Not the direction I thought we would go with this but I like it nevertheless.

**Pull request merged**